### PR TITLE
add practice and prerequisites - 2

### PR DESCRIPTION
**Pull request by valentin-p**
- Adds the practices  and prerequisites for 6 practice exercices
- Relates to #1476

### CODE REVIEW

**Review by ErikSchierboom — COMMENTED** [review]
Excellent work! I've left a comment, but once that is fixed, this is ready to merge! Thanks!

**Comment by valentin-p** [comment]
I could not find the comment.

**Comment by ErikSchierboom** [comment]
Weird, I'll re-add it.

File: config.json [hunk 1]
```diff
@@ -671,8 +671,12 @@
         "slug": "gigasecond",
         "name": "Gigasecond",
         "uuid": "b7116dfb-1107-4546-9f95-f15acdb6f6a4",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "datetimes"
+        ],
+        "prerequisites": [
+          "numbers"
```
Comment (ErikSchierboom) [inline config.json:678]: Currently, the `practices` and `prerequisites` values don't overlap at all. This is usually not what we want. Take this `gigasecond` exercise. `datetimes` is listed in the `practices` field, which is 100% correct. However, as Practice Exercises are not intended to teach Concepts but to help them practice a Concept that they've already learnt. Therefore, students should already be familiar with the `datetimes` Concept before being able to work on `gigasecond`. Thus, the `datetimes` Concept must also be a prerequisite:

```suggestion
          "datetimes",
          "numbers"
```

This comment also applies to the other exercises.

**Comment by ErikSchierboom** [comment]
@valentin-p Done

**Comment by valentin-p** [comment]
Updated.

**Comment by ErikSchierboom** [comment]
@valentin-p Great, but the other exercises should be updated too :)

**Comment by valentin-p** [comment]
Sure. added all practices as prerequisites too

File: config.json [hunk 2]
```diff
@@ -715,8 +740,15 @@
         "slug": "hamming",
         "name": "Hamming",
         "uuid": "05162ee0-38ac-40ad-a591-d70a74b6963a",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "if-statements",
+          "for-loops"
```
Comment (ErikSchierboom) [inline config.json:745]: Maybe also add `linq`?

```suggestion
          "for-loops",
          "linq"
```

File: config.json [hunk 3]
```diff
@@ -727,8 +759,15 @@
         "slug": "nucleotide-count",
         "name": "Nucleotide Count",
         "uuid": "caca1c6a-b998-431e-b6af-ca2d47b7708f",
-        "practices": [],
-        "prerequisites": [],
+        "practices": [
+          "dictionaries"
```
Comment (ErikSchierboom) [inline config.json:763]: Maybe also add `linq`?

```suggestion
          "dictionaries",
          "linq"
```

**Review by ErikSchierboom — APPROVED** [review]
Looks good! As a side note: it is entirely possible that the `practices` key contains values that are not in the `prerequisites` key. I've even suggested to examples where that is the case :)

File: config.json (hunk 2 above)
Comment (valentin-p) [reply config.json:745]: I though about that one, but didn't know if it was allowed.

**Comment by ErikSchierboom** [comment]
Awesome work! 🎉

**Pull request merged**